dockers/release-rocm/Dockerfile

```diff
@@ -0,0 +1,40 @@
+ARG PYTHON_VERSION=3.9
```

Comment:
I think we shouldn't add this for now because
1. If I see it correctly, this dockerfile isn't specific to rocm at all, so we can simply reuse `dockers/release/Dockerfile`, and
2. we won't release docker image for ROCm anyway until PL officially supports it.

Reply:
We removed the release-rocm folder and just shifted the Dockerfile from there into the base-rocm folder with a different file name